src/main/java/org/springframework/data/redis/connection/lettuce/LettucePoolingClientConfiguration.java

```diff
@@ -32,10 +32,12 @@
  * @author Mark Paluch
  * @author Christoph Strobl
  * @author Yanming Zhou
+ * @author Asmir Mustafic
  * @since 2.0
  */
 public interface LettucePoolingClientConfiguration extends LettuceClientConfiguration {
 
+	boolean preparePool();
```

Comment:
is by your BC policy a compatibility break? how to add an additional configuration?

Reply:
Adding a method to an interface is formally a breaking change if the new method is not a `default` one. However, we also distinguish between interfaces implemented by our framework and those, that are intended to be implemented by applications. The config infrastructure is to hide implementation details and so that would have been a safe change to make.

I would not expose this flag at all, because the functionality to prepare the pool should be part of the pool initialization.

Reply:
I have handled it as a bug and removed it